fe/fe-core/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -735,9 +735,17 @@
      */
     @ConfField(mutable = true, masterOnly = true)
     public static int max_backend_down_time_second = 3600; // 1h
+
+    /**
+     * If disable_storage_medium_check is true, ReportHandler would not check tablet's storage medium
+     * and disable storage cool down function, the default value is false.
+     * You can set the value true when you don't care what the storage medium of the tablet is.
+     */
+    @ConfField(mutable = true, masterOnly = true)
+    public static boolean disable_storage_medium_check = false;
```

Comment:
enable_strict_storage_medium_check make the user confused, if we really need to check storage_medium, we should not create table without consider storage medium and then produce many migration task usually failed because that user not use this feature. If we not care the storage medium, just disable this check, which can save a lot of time for tablet report, because it is meanless to distinguish ssd from hdd for some users while user not need cool down feature.